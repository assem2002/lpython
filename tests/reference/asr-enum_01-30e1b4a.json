{
    "basename": "asr-enum_01-30e1b4a",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/enum_01.py",
    "infile_hash": "5f229c1dc30fb06044461359242b2a63d9b93c4086669b391cef7426",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-enum_01-30e1b4a.stderr",
    "stderr_hash": "ffe4d855c9b53ad57a7d2c709c9279f1a49b97e6e5c1c88c15b0a578",
    "returncode": 2
}