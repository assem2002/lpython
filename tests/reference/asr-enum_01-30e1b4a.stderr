semantic error: Enumerations with non-integer or non-unique integer values cannot interoperate with C code.
 --> tests/errors/enum_01.py:5:1 - 8:19
  |
5 |    class Color(Enum):
  |    ^^^^^^^^^^^^^^^^^^...
...
  |
8 |        BLUE: f64 = 0.7
  | ...^^^^^^^^^^^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
